js/utils/templateHelpers.js

```diff
@@ -1,7 +1,22 @@
 import app from '../app';
+import { getHiRez } from './responsive';
+import { getLargeWidth } from './responsive';
+import { getSmallHeight } from './responsive';
 
 export function polyT(...args) {
   return app.polyglot.t(...args);
 }
 
 export const getServerUrl = app.getServerUrl;
+
+export function hiRez() {
```

Comment:
No particular reason for the name change, just incomplete refactoring from earlier code. I'll change it.